### PR TITLE
Plat 1619 revisit exception logging

### PR DESCRIPTION
**Pull request by andela-sjames**
PR: https://themuse.atlassian.net/browse/PLAT-1619

Adds a generic encoder condition that captures all python classes.

### CODE REVIEW

File: muselog/datadog.py [hunk 1]
```diff
@@ -72,8 +72,15 @@ def default(self, obj):
             return "traceback"
         elif isinstance(obj, timedelta):
             return obj.__str__()
-
-        return super().default(obj)
+        else:
```
Comment (ian-axelrod) [inline muselog/datadog.py]: Is there a library that provides a very robust JSONEncoder? I know this is an issue that plagues many python devs, so I suspect someone has created a library that can encode /most/ stdlib objects. This wouldn't eliminate the need for a fallback like you have, but it should be a marked improvement.

Comment (andela-sjames) [reply muselog/datadog.py]: I will check further.

Comment (andela-sjames) [reply muselog/datadog.py]: I couldn't find any custom library for python JSON encoder, as most scripts just try to take care of specific types to encode, similar to what we do too.

Comment (ian-axelrod) [reply muselog/datadog.py]: Okay.

Comment (Shikkic) [reply muselog/datadog.py]: I was searching earlier and couldn't find much either, it's lame

**Comment by Shikkic** [comment]
@andela-sjames Can you bump the version real quick

**Pull request merged**